### PR TITLE
Feature / DAO-284

### PR DESCRIPTION
**Pull request by Fabricevladimir**
Updated DAO settings page to match the new designs

- Changed path from `daos/:daoName/dao-settings` to `daos/:daoName/settings`

### CODE REVIEW

File: packages/govern-console/src/containers/DAO/DaoHomePage.tsx [hunk 1]
```diff
@@ -18,7 +19,7 @@ const DaoHomePage: React.FC = () => {
         <Redirect exact from={path} to={`${path}actions`} />
         <ApmRoute path={`${path}actions`} render={() => <div>Actions</div>} />
         <ApmRoute path={`${path}finance`} render={() => <div>Finance</div>} />
-        <ApmRoute path={`${path}settings`} render={() => <div>Settings</div>} />
+        <ApmRoute path={`${path}settings`} render={() => <DaoSettings />} />
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAO/DaoHomePage.tsx:22]: Two notes here:

1. Currently, the path `daos/:daoName/dao-settings` (defined in `Homepage.tsx`) is still in use. This should eventually get deleted, right?
2. The APM route component is called in two different ways here and in `HomePage.tsx` (here, `render` prop is used, there, `component`prop). Is there a particular reason for that?

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx [hunk 2]
```diff
@@ -60,58 +49,58 @@ interface FormInputs {
...
+
   const methods = useForm<FormInputs>();
   const { control, setValue, getValues, handleSubmit, trigger } = methods;
-  const { daoName } = useParams<ParamTypes>();
-  //TODO daoname empty handling
-  const { data: dao, loading: loadingDaoData } = useDaoQuery(daoName);
-  const [daoDetails, updateDaoDetails] = useState<any>();
+
+  const context: any = useWallet();
+  const { account, isConnected, provider } = context;
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]:
```suggestion
  const { account, isConnected, provider } = useWallet();
```

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx [hunk 3]
```diff
@@ -243,294 +242,65 @@ const DaoSettings: React.FC = () => {
     });
   };
 
+  /**
+   * Render
+   */
   return (
-    <PageContent card={<SettingsCard />}>
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx:247]: Why is this removed (and not replaced)? The `SettingsCard` component is still part of the design on Figma.

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx [hunk 4]
```diff
@@ -243,294 +242,65 @@ const DaoSettings: React.FC = () => {
...
-              />
-            </div>
-          </FormProvider>
+    <Box>
+      <Grid gap={SPACING[layoutName]}>
+        <div>
+          <StyledText name={'title1'}>Settings</StyledText>
+          {/* TODO: Proper copy needs to be added to the subtitle */}
+          <StyledText name={'title4'} style={{ color: disabledContent }}>
+            Message about the importance of permissions on your DAO, bla bla, explaining the
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]: I guess this copy is not your fault :joy:

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx [hunk 5]
```diff
@@ -243,294 +242,65 @@ const DaoSettings: React.FC = () => {
...
-                label={'Schedule configuration changes'}
-                buttonType={'primary'}
-                onClick={handleSubmit(callSaveSetting)}
-                wide
-              />
-            </div>
-          </FormProvider>
+    <Box>
+      <Grid gap={SPACING[layoutName]}>
+        <div>
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]: I think this div may need a bit more margin at the bottom.

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx [hunk 6]
```diff
@@ -243,294 +242,65 @@ const DaoSettings: React.FC = () => {
...
+          <IPFSInput
+            rows={4}
+            title="Justification for change of settings"
+            subtitle="Insert the reason for scheduling this change of settings, so DAO members can understand it."
+            placeholder="Please insert the reason why you want to execute this"
+            textInputName="proof"
+            fileInputName="proofFile"
+          />
+          <div>
+            <Button
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]: Button should be a bit taller I think.

File: packages/govern-console/src/containers/DAOSettings/components/SettingsCard.tsx [hunk 7]
```diff
@@ -1,3 +1,7 @@
+/**
+ * TODO: Unused component. To be removed.
```
Comment (WissenIstNacht) [inline packages/govern-console/src/containers/DAOSettings/components/SettingsCard.tsx:2]: Again, are you sure?  😅

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Thanks for the refactoring! Looking clean 😏 

Please get back to me on the comments about routing and about the setting page's side card stuff.

File: packages/govern-console/src/containers/DAO/DaoHomePage.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/containers/DAO/DaoHomePage.tsx:22]: The final routing setup for these pages (Actions, Finance, Settings) is being done on the feature/dao-side-card branch. 
1. Yes, it's going to be removed. 
2. The reason we're using the render props pattern here is because we will be passing down the list of actions (proposals, transactions) to the Actions component. All other components that don't need props from the `DaoHomePage` will be using the `component` prop as apposed to `render`.

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx (hunk 3 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx:247]: The `SettingsCard` is removed in the new designs... We're only showing the `HelpComponent`

File: packages/govern-console/src/containers/DAOSettings/components/SettingsCard.tsx (hunk 7 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/containers/DAOSettings/components/SettingsCard.tsx:2]: Yes! 😄

File: packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]: This one is a type issue. I'll try to modify the return type of `useWallet`. Currently, it's only an empty object and so this workaround had to be used to silence the TS compiler.

Comment (WissenIstNacht) [reply packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx]: Sorry, I didn't realize you were dealing with an extended use-wallet. Thanks for finding a workaround in the augmented wallet provider, though 👍🏼

**Review by WissenIstNacht — APPROVED** [review]
